dev/com.ibm.ws.kernel.boot.ws-server/publish/bin/server

```diff
@@ -919,7 +922,16 @@ javaCmd()
       CHECKPOINT_AT="features"
     fi
     REMAINING_ARGS="$REMAINING_ARGS --internal-checkpoint-at=$CHECKPOINT_AT"
-    JVM_OPTIONS_QUOTED="$JVM_OPTIONS_QUOTED -XX:+EnableCRIUSupport -Dosgi.bundlefile.limit=0"
+
+    # Default list of immutable env vars for checkpoint
+    X_WLP_IMMUTABLE_VARS="-Dorg.eclipse.openj9.criu.ImmutableEnvVars=INVOKED,WLP_USER_DIR,WLP_OUTPUT_DIR,LOG_DIR,X_LOG_DIR,LOG_FILE,X_LOG_FILE,VARIABLE_SOURCE_DIRS,X_CMD"
+
+    # Add any additional user specified variables
+    if [ -z "$WLP_IMMUTABLE_VARS" ]; then
```

Comment:
should this be 
    [ -n "$WLP_IMMUTABLE_VARS"]

Reply:
Done